src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.cs

```diff
@@ -3497,6 +3756,146 @@ public new abstract class Arm64 : ArmBase.Arm64
         /// </summary>
         public static Vector64<float> FusedMultiplySubtractNegatedScalar(Vector64<float> acc, Vector64<float> left, Vector64<float> right) => FusedMultiplySubtractNegatedScalar(acc, left, right);
 
+        /// <summary>
+        /// uint8x8_t vset_lane_u8 (uint8_t a, uint8x8_t v, const int lane)
+        ///   A32: VMOV.8 Dd[lane], Rt
+        ///   A64: INS Vd.B[lane], Wn
+        /// </summary>
+        public static Vector64<byte> Insert(Vector64<byte> vector, byte index, byte data) => Insert(vector, index, data);
+
+        /// <summary>
+        /// float64x1_t vset_lane_f64 (float64_t a, float64x1_t v, const int lane)
+        ///   A32: VMOV.F64 Dd, Dm
+        ///   A64: INS Vd.D[lane], Vn.D[0]
+        /// </summary>
+        public static Vector64<double> Insert(Vector64<double> vector, byte index, double data) => Insert(vector, index, data);
```

Comment:
Same here, not sure this API makes sense, it can just be `Vector64.CreateScalar(double)` since the `index` can only ever be `0` and the `vector` doesn't matter

Reply:
or maybe just expose as `public static Vector64<double> Insert(double data)`

Reply:
I will remove - I probably lost a commit removing those in one of my rounds of rebase

Reply:
Or alternatively - we can name these as 
```c#
public static Vector64<double> InsertScalar(double data);
public static Vector64<long> InsertScalar(long data);
public static Vector64<ulong> InsertScalar(ulong data);
```
to make them slightly more distinguishable from `Insert` that accept `index`?

cc @TamarChristinaArm @CarolEidt

Reply:
I'm with @tannergooding here that we should just have this one be `CreateScalar` is we don't want the index parameter.

Reply:
Okay, I will remove `Insert` for these types and implement CreateScalar as a separate PR.